src/custom-types.json

```diff
@@ -878,7 +887,12 @@
     "compressible": false
   },
   "video/webm": {
-    "compressible": false
+    "compressible": false,
+    "extensions" : ["webm"],
```

Comment:
So this is already in NGINX and Apache, and as such, this is already in our database. Adding it here does nothing. Are you having an issue where you believe adding the extension will resolve some kind of issue, and if so, what is the issue you're having?

Reply:
Oh. I thought I had issues with webm. Apparently not. I will remove this.